src/sentry/search/snuba/executors.py

```diff
@@ -453,6 +461,24 @@ def trend_aggregation(start: datetime, end: datetime) -> Sequence[str]:
     ]
 
 
+def better_priority_aggregation(
+    start: datetime, end: datetime, age: int, log_level: int, frequency: int, has_stacktrace: int
+) -> Sequence[str]:
+    event_age = "divide(now() - timestamp, 3600)"
```

Comment:
I wonder if it would be better to use an arbitrary reference point in time, say the time of sentry's first commit, instead of `now()` (similar to reddit's ranking works https://www.quora.com/Where-do-the-constants-1134028003-and-45000-come-from-in-reddits-hotness-algorithm). 

That way each event and issue age stays static and isn't dependent on the time when the query executes. This has a nice benefit where the priority score stays the same if no new events group to the issue. If we use `now()` the score constantly changes and its harder to compare why X issue ranks better than Y issue unless we aggregate them both in the same query.

Reply:
Using forward time decay like this is usually most useful if you're attempting to avoid recalculating the scores for the whole list every time, since like you said the score won't decrease over time, but instead new events are worth exponentially more over time (and then we avoid making extremely large numbers by using `log`).

Since we're always calculating these on the fly there's less value. I agree that there's some value in being able to use it as a point of comparison, but I'm not sure that it's worth complicating the formula we have at the moment.